### PR TITLE
Updated mkdocs base image

### PR DESCRIPTION
**Pull request by benarent**
This PR updates our base image for mkdocs. The big upgrades are

- Bump 1.x for MKDocs https://www.mkdocs.org/about/release-notes/#version-10-2018-08-03 
- Bump [Markdown](https://python-markdown.github.io/) from 2x to 3x https://python-markdown.github.io/change_log/release-3.1/ 
- Bump [Click](https://pypi.org/project/click/) to 7.0 
- Add https://pypi.org/project/mkdocs-markdownextradata-plugin/ 

I removed some old draw.io images and combined the README with the intro.

### CODE REVIEW

File: docs/4.2/enterprise/ssh_fips.md [hunk 1]
```diff
@@ -136,6 +138,12 @@ cryptographic module (BoringCrypto) and fails to start if it was not.
 * If recording proxy mode is selected, validation of host certificates should always happen.
 
 
+### FedRAMP Audit Log
+
+. At the close of a connection (close of a *srv.ServerContext) the total data transmitted and received is emitted to the Audit Log.
```
Comment (webvictim) [inline docs/4.2/enterprise/ssh_fips.md]: Full stop at the start here? Should probably be `*`

File: docs/4.2/enterprise/ssh_fips.md [hunk 2]
```diff
@@ -136,6 +138,12 @@ cryptographic module (BoringCrypto) and fails to start if it was not.
...
+
+
+
```
Comment (webvictim) [inline docs/4.2/enterprise/ssh_fips.md]: Extra space

File: docs/4.1.yaml [hunk 3]
```diff
@@ -9,20 +9,22 @@ copyright: Gravitational Inc, 2016-19
...
-theme_dir: theme
+theme:
+    name: null
+    custom_dir: 'theme/'
 markdown_extensions:
   - markdown_include.include
   - admonition
   - def_list
   - footnotes
+  #- codehilite
```
Comment (webvictim) [inline docs/4.1.yaml]: Commented out?

File: docs/4.1/quickstart.md [hunk 4]
```diff
@@ -100,15 +100,15 @@ We've got Teleport running but there are no users recognized by Teleport Auth
...
+!!! info "OS User Mappings"
     The OS user `teleport` must exist! On Linux, if it
     does not already exist create it with `adduser teleport` . If you do not have
     the permission to create new users on the VM, run `tctl users add teleport
     <your-username> ` to explicitly map ` teleport` to an existing OS user. If you
     do not map to a real OS user you will get authentication errors later on in
     this tutorial!
 
-``` bash
+```
```
Comment (webvictim) [inline docs/4.1/quickstart.md]: This is no longer bash?

**Review by webvictim — COMMENTED** [review]
Lots of red in this diff - all dead code?

File: docs/4.1.yaml (hunk 3 above)
Comment (benarent) [reply docs/4.1.yaml]: ow, yeah. Removed for now. Will open another PR with updates to our code highlighting.

File: docs/4.1/quickstart.md (hunk 4 above)
Comment (benarent) [reply docs/4.1/quickstart.md]: Strange, I've fixed that.

**Review by webvictim — APPROVED** [review]
LGTM assuming `milv` has nothing to report :)

**Comment by benarent** [comment]
Extra Notes:
When testing on staging I found that sidebar links didn't work, I've fixed this. The Theme switcher had the same redirect issue . I've hacked this fix for now https://github.com/gravitational/teleport/pull/3314/commits/6f3667454f900e0e0ea912dab29b09e09543fdaa but we'll want to fix this. 

I've also removed the strange '/index.html' page redirect thing that me and @russjones looked it. One less redirect 🙌 .

**Pull request merged**